docs/Dialects/FIRRTL/FIRRTLIntrinsics.md

```diff
@@ -289,3 +289,94 @@ always @(posedge clock) begin
   end
 end
 ```
+
+### circt_view
+
+This will become a SystemVerilog Interface that is driven by its arguments.
+This is _not_ a true SystemVerilog Interface, it is only lowered to one.
+
+
+| Parameter     | Type   | Description                                         |
+| ------------- | ------ | --------------------------------------------------- |
+| name          | string | Instance name of the view.                          |
+| view          | string | JSON encoding the view structure.                   |
+
+| Port       | Direction | Type     | Description                         |
+| ---------- | --------- | -------- | ----------------------------------- |
+| ...        | input     | Ground   | Leaf ground values for the view     |
+
+The structure of the view is encoded using JSON, with the top-level object
+required to be an `AugmentedBundleType`.
+
+The intrinsic operands correspond to the `AugmentedGroundType` leaves,
+and must be of ground type.
+
+This encoding is a trimmed version of what's used for the old GrandCentral View
+annotation.
+
+These views do not interact with any GrandCentral annotations or options governing
+the behavior of the annotation-based views.
+
+Output directory behavior is a work in progress presently.
+
+#### AugmentedGroundType
+
+| Property   | Type   | Description                                          |
+| ---------- | ------ | -------------                                        |
+| class      | string | `sifive.enterprise.grandcentral.AugmentedGroundType` |
+
+Creates a SystemVerilog logic type.
+
+Each ground type corresponds to an operand to the view intrinsic.
+
+Example:
+```json
+{
+  "class": "sifive.enterprise.grandcentral.AugmentedGroundType"
+}
+```
+
+#### AugmentedVectorType
+
+| Property   | Type   | Description                                          |
+| ---------- | ------ | -------------                                        |
+| class      | string | `sifive.enterprise.grandcentral.AugmentedVectorType` |
+| elements   | array  | List of augmented types.
```

Comment:
```suggestion
| elements   | array  | List of augmented types. |
```

Reply:
Eep.  This is present in the annotation form as well.